modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -26,6 +26,7 @@ namespace LORIS\conflict_resolver;
  */
 class Conflict_Resolver extends \NDB_Menu_Filter_Form
 {
+    public $skipTemplate = true;
```

Comment:
we can remove this line if the smarty template is deleted

Reply:
I get this error message below if I remove the $skipTemplate:
The following errors occurred while attempting to display this page:
Unable to load template 'file:menu_conflict_resolver.tpl'
If this error persists, please report a bug to your administrator .

Reply:
@zaliqarosli for your comment "i agree with henri. we should be passing states through props" here is the reason. I did it with an outside function because of the two components and from what I've read online you can only set the props once. I'm passing data between components and to update the state of another component.